encord/orm/project.py

```diff
@@ -149,10 +149,48 @@ class ProjectDataset:
     pass
 
 
-class ProjectCopyOptions(Enum):
+class ProjectCopyOptions(str, Enum):
     COLLABORATORS = "collaborators"
     DATASETS = "datasets"
     MODELS = "models"
+    LABELS = "labels"
+
+
+class ReviewApprovalState(str, Enum):
+    APPROVED = "APPROVED"
+    PENDING = "PENDING"
+    REJECTED = "REJECTED"
+    DELETED = "DELETED"
+    NOT_SELECTED_FOR_REVIEW = "NOT_SELECTED_FOR_REVIEW"
+
+
+@dataclass
+class CopyProjectMetadata:
+    project_title: str
+    project_description: str
+    dataset_title: Optional[str] = None
+    dataset_description: Optional[str] = None
+
+
+@dataclass
+class CopyReviewTasksOptions:
+    copy_review_tasks: bool = False
+    review_status_list: List = field(default_factory=list)
```

Comment:
What would happen if you have `True` and then some items in this list? If it wouldn't make sense, can we define a type that only allows options that do make sense?